app/models/chunk.rb

```diff
@@ -10,6 +10,10 @@ class Chunk < ApplicationRecord
   scope :embedded, -> { where.not(vector_id: nil) }
   scope :extracted, ->  { where(entities_extracted: true) }
 
+  # TODO: Take out next 2 lines if not needed.
+  # delegate :link, to: :document
+  # delegate :filename, to: :document
+
```

Comment:
My rule of thumb is not to add speculative stuff, even if it looks like it might be handy. So, might as well take them out?